build.sbt

```diff
@@ -8,7 +8,9 @@ import scala.sys.process.Process
 import scala.util.Try
 
 // Import general settings from Silver, Silicon and Carbon
-lazy val silver = project in file("silver")
+
+// we assume that carbon/silver and silicon/silver point to the same version of the silver repo
+lazy val silver = project in file("silicon/silver")
```

Comment:
I'm fine with this assumption for now, while we do not come up with a better solution for viperserver. However, to avoid any surprises, I would probably add a check here similar to the one we do on the Dockerfile (we could even eval the result of the same git command using something like [this](https://alvinalexander.com/scala/how-to-execute-external-commands-stdout-scala/))

Reply:
I chose not to include such a check in SBT (neither in Gobra nor ViperServer) because I think this might interfere with whatever local setup you're using. Maybe you want to still rely on sym links for development purposes or you know what you're doing or ...

Reply:
But would the project even compile if you try to use different versions of the dependencies?

Reply:
At any rate, feel free to merge it